book/controller.rst

```diff
@@ -469,31 +465,48 @@ perform a 301 (permanent) redirect, modify the second argument::
 Rendering Templates
 ~~~~~~~~~~~~~~~~~~~
 
-If you're serving HTML, you'll want to render a template. The ``render()``
-method renders a template **and** puts that content into a ``Response``
-object for you::
+To serve HTML, template needs to be rendered and the content put into
+the ``Response`` object. The shortcut method
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::render`
+of ``Controller`` class does just that::
```

Comment:
I like a lot of the ways you re-phrased things, but not this one :). I'm reverting back to the original, which I still think is quite clear.